### DISCUSSION TITLE
SSR: window is not defined

### DISCUSSION THREAD

**Discussion by Skywalker-72** [codex-team/editor.js#1783] [General]
There is an open issue about this problem : https://github.com/codex-team/editor.js/issues/1036#

**Comment by Skywalker-72** [comment]
I have wrapped my EditorJS initialization in isPlatformBrowser()
```
if(isPlatformBrowser(this.platformId)){
            this.editor = new EditorJS({
                holder: 'editorjs'
            })
 }
```
I have mocked the window Object in my node server using domino
```
const template = readFileSync(join(distFolder, indexHtml)).toString();
const win = domino.createWindow(template.toString());

global['window'] = win;
import 'zone.js/node';
import { AppServerModule } from './src/main.server';
import { APP_BASE_HREF } from '@angular/common';
```
I still see the `window is not defined` error. 
My Angular version: 10
EditorJS version: 2.22.2

**Comment by Skywalker-72** [comment]
@Teebo this is me again. Pardon, I was using my office account for the discussion earlier. 
May I know on what Angular version were u trying the solution

**Reply by Teebo** [reply]
@Skywalker-72 it is version 9.0.3

**Reply by Skywalker-72** [reply]
Thanks
